templates/expo-template-bare-minimum/ios/Podfile

```
@@ -22,6 +22,16 @@ target 'HelloWorld' do
   use_frameworks! :linkage => podfile_properties['ios.useFrameworks'].to_sym if podfile_properties['ios.useFrameworks']
   use_frameworks! :linkage => ENV['USE_FRAMEWORKS'].to_sym if ENV['USE_FRAMEWORKS']
 
+  # Flipper support in expo via build properties plugin
+  expo_flipper = FlipperConfiguration.disabled
+  if ENV['FLIPPER_DISABLE'] != "1" && ENV['NO_FLIPPER'] != "1" && defined?(podfile_properties['ios.flipper']) then
+    if podfile_properties['ios.flipper'] == true then
+      expo_flipper = FlipperConfiguration.enabled
+    elsif podfile_properties['ios.flipper'] != false then
```

```suggestion
    elsif podfile_properties['ios.flipper'] == "false" then
      ENV['NO_FLIPPER'] = '1'
      expo_flipper = FlipperConfiguration.disabled
    else
```

i would recommend rewrite the disabled logic to overwrite `ENV['NO_FLIPPER']` because now upstream told people to leverage the `NO_FLIPPER` for 3rd party plugin integration: https://github.com/facebook/react-native/blob/208f5595055426305a9f23e92546b2ad09a8a52c/template/ios/Podfile#L7-L15